crossbar/common/checkconfig.py

```diff
@@ -1544,6 +1544,29 @@ def check_listening_transport_web(transport, with_endpoint=True):
 _WEB_PATH_PATH = re.compile(_WEB_PATH_PAT_STR)
 
 
+def check_listening_transport_mqtt(transport):
```

Comment:
[Seems](http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/os/mqtt-v3.1.1-os.html#_Toc398718028) like the first octet is `0x10`, right?

* `0x10` is a LF in ASCII, and I think this cannot be a valid first octet for any HTTP request? At least practically ..
* `0x7F' is first octet of RawSocket

So we probably could make it work (I mean, being able to run MQTT on a standalone transport port *and* being able to run it under universal transport switching).

The thing with that is: one could then use 1 TLS certificate for everything .. which is nice.